torch/_inductor/codegen/rocm/ck_universal_gemm_template.py

```diff
@@ -380,6 +545,67 @@ def render(self, kernel: ROCmTemplateKernel, op: "CKGemmOperation", **kwargs) ->
             version_comment=version_comment,
         )
 
+        if config.rocm.generate_test_runner:
+            is_static_problem = all(is_static_int(arg) for arg in self.size_args())
+            M, N, K, LDA, LDB, LDC, LDD = (
+                self.size_args()
+                if is_static_problem
+                else (
+                    f"std::stoi(argv[{k}])" for k, _ in enumerate(self.size_args(), 1)
+                )
+            )
+            has_bias = Bias is not None
+            has_scale = scale_x is not None and scale_w is not None
+            runner_code = self._template_from_string(
+                self.standalone_runner_template
+            ).render(
+                inline_utils=self.inline_utils().getvalue(),
+                kernel_name=kernel.kernel_name,
+                M=M,
+                N=N,
+                K=K,
+                LDA=LDA,
+                LDB=LDB,
+                LDC=LDC,
+                LDD=LDD,
+                has_bias=has_bias,
+                has_scale=has_scale,
+                a_ck_dtype=op.a_element_dtype,
+                b_ck_dtype=op.b_element_dtype,
+                c_ck_dtype=op.c_element_dtype,
+                bias_ck_dtype=op.ds_element_dtypes[0] if has_bias else "",
+                scale_a_ck_dtype=op.ds_element_dtypes[0]
+                if has_scale and 2 == len(op.ds_element_dtypes)
+                else "BF16",
+                scale_b_ck_dtype=op.ds_element_dtypes[1]
+                if has_scale and 2 == len(op.ds_element_dtypes)
+                else "BF16",
+                a_torch_dtype=DTYPE_TO_CPP[X.get_layout().dtype],
+                b_torch_dtype=DTYPE_TO_CPP[W.get_layout().dtype],
+                c_torch_dtype=DTYPE_TO_CPP[Y.get_layout().dtype],
+                bias_torch_dtype=DTYPE_TO_CPP[Bias.get_layout().dtype]
+                if Bias is not None
```

Comment:
can use `has_bias` here =p

Reply:
mypy can't infer that Bias is not None when the method is accessed, and linter job fails